wordpress-mobile/WordPress-iOS
# Issues/9625 zendesk unlocalized plans

Fixes #9625

This PR fetches and caches plan info for non-en locales and includes that info in new Zendesk issues. 


To test:
In Xcode, edit the scheme and set the device language to something other than English. 
Launch the branch. 
Go to me > Help & Support > Contact Us
Create a new ticket. 

Log into Zendesk and find your ticket. 
Check the meta data for blogs. 
Confirm that there is additional non-localized plan information. 

Repeat this test with the device set to English. 
Confirm that the additional information is not included with the ticket. 


Update release notes:
